core/includes/updater.inc

```diff
@@ -186,8 +186,11 @@ abstract class Updater implements BackdropUpdaterInterface {
    * @throws UpdaterException
    */
   public static function getProjectType($directory) {
-    if ($directory == BACKDROP_ROOT) {
-      return 'core';
+    if (basename($directory) == 'backdrop') {
```

Comment:
I think this will only work if the directory backdrop is installed in is named `backdrop` if for example you install backdrop in a directory named `www` this will fail.

Reply:
Nope, $directory is the stream wrapper to the extracted zip file (temporary://) and NOT the install dir. ;)